bucket/covscript.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": "3.3.2",
+    "description": "Covariant Script interpreter",
+    "homepage": "https://covscript.org",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/covscript/covscript/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/covscript/covscript/releases/download/3.3.2/covscript-win64.7z",
+            "hash": "c1b250bbc995982c33f1e33fefd62a794626d131e71c116145b23a1d2bd076f8",
+            "extract_dir": "build_x64"
+        },
+        "32bit": {
+            "url": "https://github.com/covscript/covscript/releases/download/3.3.2/covscript-win32.7z",
+            "hash": "ddc319251260bc3aab35c931c6e7761a1d8499839fb9965655653999c3de0f27",
+            "extract_dir": "build"
+        }
+    },
+    "bin": [
+        "bin\\cs.exe",
+        "bin\\cs_dbg.exe",
+        "bin\\csbuild.bat",
```

Comment:
Is this correct? Should it be used in some specific folder?

![A](https://i.imgur.com/RPryEwC.png)

Reply:
Yes. This is correct. It should be considered an "upstream issue".
Sorry that I did not mention this in the comments.

Reply:
Do you have issue id or link for it?